src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
+  static alive = [];
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
   // write your code here
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health, hidden) {
```

Comment:
The constructor of the Herbivore class should not accept a 'health' parameter since it defaults to 100 in the Animal class. Also, the 'hidden' parameter is unnecessary because it is always initialized to 'false' in the body of the constructor.